specs/checkbox.md

```diff
@@ -0,0 +1,128 @@
+# FAST-Checkbox
```

Comment:
Can we just call this "Checkbox"? We seem to be diverging among us on how to title these specs.